docs/release-notes/RELEASE-NOTES-4.2.0.md

```diff
@@ -0,0 +1,112 @@
+# About this release
+
+[This][this-release] is a maintenance release.
+It includes the following changes:
+* Localnet setup (docker-compose) has been improved by adding node API compatibility of the internal endpoints,
```

Comment:
@lucafavatella I forgot to add a note about docker-compose configs:

- Localnet (docker-compose) configuration has ben moved out to it's own repository https://github.com/aeternity/localnet